tools/coverage/main.go

```diff
@@ -48,16 +49,17 @@ func main() {
 	artifactsDir := flag.String("artifacts", defaultArtifactsDir, "directory for artifacts")
 	coverageTargetDir := flag.String("cov-target", defaultCoverageTargetDir, "target directory for test coverage")
 	coverageProfileName := flag.String("profile-name", defaultCoverageProfileName, "file name for coverage profile")
+	noRepoConn := flag.Bool("no-repo-connection", true, "enable it when connection to GibHub is not required")
 	githubTokenPath := flag.String("github-token", "", "path to token to access github repo")
-	covThresholdFlag := flag.Int("cov-threshold-percentage", defaultCovThreshold, "token to access github repo")
+	covThresholdFlag := flag.Int("cov-threshold-percentage", defaultCovThreshold, "token to access GibHub repo")
```

Comment:
```suggestion
	covThreshold := flag.Int("cov-threshold-percentage", defaultCovThreshold, "token to access GibHub repo")
```

Reply:
addressed in https://github.com/knative/test-infra/pull/1425